app/models/user.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'User',
+    {
+      id: DataTypes.INTEGER,
+      firstName: DataTypes.STRING,
+      lastName: DataTypes.STRING,
+      email: DataTypes.STRING,
+      password: DataTypes.STRING
```

Comment:
you should add the timestamps here 

`      
createdAt: { allowNull: false, type: DataTypes.DATE },
updatedAt: { allowNull: false, type: DataTypes.DATE },
`